src/assets/questionsAndAnswers/questionsAndAnswers.js

```diff
@@ -135,7 +135,7 @@ export default {
                     {
                         "id": "useQuestion1",
                         "question": "What does this map show me?",
-                        "answer": "The maps shows each season's average daily water use rate of withdrawal for each watershed during 2015 divided into 3 month periods for thermoelectric, irrigation, and public supply categories of use. Original modeled data are available at watershed units or 12-digit hydrologic unit code (HUC), USGS 2014, that were aggregated to larger 10-digit HUC for ease of display at a National scale.  The locations of dots on the map do not necessarily represent the exact location of the withdrawals."
+                        "answer": "The maps shows each season's average daily water use rate-of-withdrawal for each watershed during 2015 for thermoelectric, irrigation, and public supply categories of use. Original modeled data are available at watershed units or 12-digit hydrologic unit code (HUC), <a href='https://www.usgs.gov/core-science-systems/ngp/national-hydrography/access-national-hydrography-products' target='_blank'>USGS 2014</a>, that were aggregated to larger 10-digit HUC for ease of display at a National scale.  The locations of dots on the map do not necessarily represent the exact location of the withdrawals."
```

Comment:
Any reason why you're not doing not doing this link the other way where we can get the link tracking from google analytics? i can fix it if you want, just wondering

Reply:
Nope, just wasn't thinking about it